src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -61,6 +61,8 @@ private static void createVoltageControls(LfNetwork lfNetwork, List<LfBus> lfBus
             List<LfGenerator> voltageControlGenerators = controllerBus.getGenerators().stream().filter(LfGenerator::hasVoltageControl).collect(Collectors.toList());
```

Comment:
Here we just have the generators linked to a controller bus. But, what happens in the case where we have two controller buses with each one SVC with slope ? We don't support that configuration too.